src/app/chip_data_model.gni

```diff
@@ -456,6 +456,11 @@ template("chip_data_model") {
           "${_app_root}/clusters/${cluster}/ArlEncoder.cpp",
           "${_app_root}/clusters/${cluster}/ArlEncoder.h",
         ]
+      } else if (cluster == "push-av-stream-transport-server") {
+        sources += [
```

Comment:
I don't think this is needed if you follow the cluster server pattern, only have {classs_name}.cpp/{classs_name}.h